build123d/common/vscode.py

```diff
@@ -0,0 +1,8 @@
+try:
+    print('OCP_VSCODE: Loading...')
+    from ocp_vscode import set_defaults, Camera, show_object
+    set_defaults(reset_camera=Camera.KEEP)
+    # TODO: Naming sidebar objects.
+    print('OCP_VSCODE: Loaded')
+except Exception as error:
```

Comment:
Replace generic exception with the specific exception(s) that this block may throw